templates/openwisp2/settings.py

```diff
@@ -51,6 +54,9 @@
 ]
 
 EXTENDED_APPS = ['django_netjsonconfig', 'django_x509']
```

Comment:
Transform this in a multiline list and insert django_netjsongraph in the list as you are doing in `INSTALLED_APPS`